config/settings.yml

```diff
@@ -401,6 +401,12 @@ flipper:
     - travis.hilton@oddball.io
     - rian.fowler@adhoc.team
     - rianfowler@outlook.com
+    - jbalboni@gmail.com
```

Comment:
@annaswims We don't have a process for updating this list but this seems fine for now. you mentioned off-boarding this morning and cleaning up this list could be part of that

Reply:
I'd like to add a few others to this list for redundancy. 

@kreek @markgreenburg @lalexanderson-dsva do you all know what your ID.me wallet email addresses are for Staging and Production environments? Can you also double check and make sure that these accounts are properly secured and do not use "throw-away" password? If you do not want to be added as an admin, just reply back to this thread with opt-out, otherwise provide:

Production: kam@adhocteam.us
Staging: kam@adhocteam.us

Reply:
on second thought -- would be nice to have this in staging asap, can add additional redundancy in a future PR.

Reply:
Do the id.me users exist for all of these users in both staging and prod?  Email validation can be bypassed in staging, which plausibly could cause an issue.